MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: MAUI
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) is a lightweight calendar control that allows user to easily select a Single, Multiple and a Range of dates. It offers Month, Year, Decade, and Century view options to quickly navigate to the desired date. To restrict the date selection by using the minimum, maximum, enable past dates, and disabled dates.
+
+
+## Key features
+
+* **Multiple calendar views**: The user can easily select and switch between the built-in month, year, decade, and century views by displaying them. Supports programmatic navigation.
+
+   {% multiple_column %}
+   {% column %}
+   ![monthview-in-maui-calendar](images/overview/MultipleViews/monthview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![yearview-in-maui-calendar](images/overview/MultipleViews/yearview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![decadeview-in-maui-calendar](images/overview/MultipleViews/decadeview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-in-maui-calendar](images/overview/MultipleViews/centuryview-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the different view modes in the SfCalendar.
+
+* **Allow view Navigation**: Allows you to navigate to the different view modes through tap interaction on the cell or header when it is enabled. Restrict users from navigating to differnt calendar views and allows you to select the dates in the year, decade and century view cells when this property is not enabled.
+
+   {% multiple_column %}
+   {% column %}
+   ![monthview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/monthview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![yearview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/yearview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![decadeview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/decadeview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/centuryview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Enable Past Dates**: Indicates whether the dates are enabled or disabled before the today date. If this property is disabled, you cannot able to select the dates before the today date and the dates before the today date are considered as disabled dates.
```

Comment:
Improve content 

Explain the feature, Don't explain value that holds( Indicates whether the dates)

Reply:
Improved enable past date content